.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,14 +19,26 @@ Optional breaking changes message. If your PR includes breaking changes. It is e
 
 ## Checklist
 
-<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+- [ ] MDX documentation adheres to Canvas Kit's [standard MDX template](https://github.com/Workday/canvas-kit/discussions/1131)
 
-- [ ] tests are changed or added
-- [ ] code has been documented
-- [ ] design approved final implementation
-- [ ] a11y approved final implementation
-- [ ] code adheres to the [API & Pattern guidelines](https://workday.github.io/canvas-kit/?path=/story/welcome-dev-docs-api-pattern-guidelines--page)
+## For the Reviewer
 
-## Additional References
+<!-- Provide a bit of context about what this PR does. -->
 
-<!-- Upload screenshots of the final component or any other artifacts that would help a reviewer understand the choices you made in the PR. -->
+## Where Should the Reviewer Start?
+
+<!-- If you were reviewing this PR, where would you want to start?  -->
+<!-- e.g. `/modules/react/common/lib/utils/someUtil.ts`  -->
+
```

Comment:
I think this might be a helpful section as well.

```suggestion

## Areas for Feedback? (optional)

<!-- Do you have any particular areas where you'd like additional focus or feedback from reviewers? -->

- [ ] Code
- [ ] Documentation
- [ ] Testing
- [ ] Codemods

<!-- If you would like to provide more context for where you'd like reviewer feedback, or if there are areas where you specifically do not want feedback, please describe below.  -->
```